### DISCUSSION TITLE
problems forking and creating a blog

### DISCUSSION THREAD

**Discussion by AxelCotonGutierrez** [community/community#57039] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to make a blog like this thema: [https://alainpham.github.io/](url)
i forked from here: [https://github.com/alainpham/alainpham.github.io](url)
to my repository and configure a few just for prove: [https://github.com/AxelCotonGutierrez/learningmathematicas](url)
the result: [https://axelcotongutierrez.github.io/learningmathematicas/](url)

why can not i see the blog right?

**Comment by airtower-luna** [comment]
Your pages use a lot of absolute links that aren't correct, one random stylesheet for example:

```html
<link rel="stylesheet" href="/assets/css/main.css" />
```

Your site is deployed to `https://axelcotongutierrez.github.io/learningmathematicas/`, so the `assets/` directory is within there, not at the top level.

The `baseurl` in your config [doesn't match either](https://github.com/AxelCotonGutierrez/learningmathematicas/blob/4bd46fe1725f79eb1823cf94f2475ea04606f409/_config.yml#L20), though oddly based on it I'd expect the link to be *differently* wrong. Still fixing that is where I'd start:

```yaml
baseurl: /learningmathematicas
```

**Reply by AxelCotonGutierrez** [reply]
i am sure is the baseurl, because i didn´t change anything else. The theme said:

baseurl: "" # the subpath of your site, e.g. /blog

and it seems that /learningmathematicas didn´t work properly either

**Comment by tshenolo** [comment]
Open your code (_includes/head.html) and replace all occurrences of /assets with assets. For example
<link rel="stylesheet" href="/assets/css/main.css" /> should be <link rel="stylesheet" href="assets/css/main.css" />

When you use /assets the browser looks for the assets folder in the parent folder, not in the learningmathematica sub folder.

Open the browser console to get better idea of errors happening. 

![image](https://github.com/community/community/assets/824557/8e76e896-8467-499a-914f-5987790a55e4)

**Reply by airtower-luna** [reply]
This is definitely the right direction. Though instead of putting the path into the template it'd probably be better to [use the `relative_url` filter](https://jekyllrb.com/news/2016/10/06/jekyll-3-3-is-here/#3-siteurl-is-set-by-the-development-server) (section 2 in the article) to adjust the path.

**Reply by AxelCotonGutierrez** [reply]
Thanks, i think a i solved most of problems, now to start changing things

Thanks again

[https://axelcotongutierrez.github.io/learningmathematicas/](url)
